web/src/pages/HomeInfo/index.tsx

```diff
@@ -225,7 +224,7 @@ function HomeInfoNetworkLogo({selectedNetwork}) {
 function HomeInfoPinnedButtons() {
   return (<>
     {/*  PINNIED BUTTONS */}
-    {false &&
+    {true &&
```

Comment:
here u have to check if there any pinned buttons. maybe add the pinned buttons to the findDefaultNetwork endpoint!

https://github.com/helpbuttons/helpbuttons/blob/main/api/src/modules/network/network.service.ts#L121

Reply:
As a first approximation I have created a new endpoint GET api/buttons/pinned.

I'm working on the layout. I'm basically missing the "pin" button click functionality. When I have that, I'll make this change.
Lots of new things. I'm going where it seems easiest or simply makes sense (for me obviously) :)

Reply:
i would put it in the default network, cause is where its only needed, and also its cached... and will keep things smoother .

Reply:
why u need to export ButtonService?